Containerfile

```diff
@@ -187,12 +189,15 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
   DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -yq --no-install-recommends \
     nodejs
 
-# Application dependencies
+# Application dependencies, for Cypress, node-canvas
 RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     --mount=type=cache,target=/var/lib/apt,sharing=locked \
     --mount=type=tmpfs,target=/var/log \
     DEBIAN_FRONTEND=noninteractive apt-get install -yq --no-install-recommends \
-      imagemagick
+      build-essential \
+      imagemagick \
+      libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb \
+      libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev
```

Comment:
I realize that `imagemagick` was already here and so this isn't net-new, but that one is already included [in Containerfile.base](https://github.com/forem/forem/blob/61f3c517e1e07020d15dc4eeb83b14c5e9b4f700/Containerfile.base#L15) and so isn't needed here.

Reply:
Good call. Let me remove it.